spring/src/test/java/org/axonframework/spring/config/annotation/SpringBeanParameterResolverFactoryTest.java

```diff
@@ -314,10 +314,11 @@ public void handle(String message, PrototypeResource resource) {
     public static class AnnotatedEventHandlerWithResources {
 
         @EventHandler
-        public void handle(String message, CommandBus commandBus, EventBus eventBus) {
+        public void handle(String message, CommandBus commandBus, EventBus eventBus, ApplicationContext applicationContext) {
```

Comment:
Would be really nice to also add a dependency that has more than one candidate and uses Spring's `@Qualifier` to resolve one. This "proves" that Spring's dependency injection mechanism is used.

Reply:
@abuijze `testMethodsAreProperlyInjected_QualifierPrecedesPrimary` should cover this case.